NSS-Day-Cohort-49/giffygram-double-rainbows
# New user sign up

#### Changes Made

1. Modified file `Register.js` to include `signUpNewUser()` 
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout new-user-sign-up
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Instructions for how reviewer can test functionality, and detailed description of what the expected outcome is.
    > Example: When sign up button is hit, you are taken to a sign up page and given the option to create a new user.
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
